### ISSUE TITLE
Increased circulating interleukin 10

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6234]
**Preferred term label:**
Increased circulating interleukin 10

**Synonyms**
Increased serum interleukin 10

**Definition (free text, please give PubMed ID)**
An elevation of the concentration of interleukin 10 in the blood circulation.
comment: IL-10 is an anti-inflammatory cytokine. During infection it inhibits the activity of Th1 cells, NK cells, and macrophages, all of which are required for optimal pathogen clearance but also contribute to tissue damage. In consequence, IL-10 can both impede pathogen clearance and ameliorate immunopathology.  PMID:18424693

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 613759    1/1, PMID:21109225

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
